tests/std/tests/GH_000625_vector_bool_optimization/test.cpp

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <algorithm>
+#include <assert.h>
+#include <vector>

```

Comment:
This new test must be added to `test.lst` or it won't be run.

Reply:
Ups